Cli-Shared/Program.cs

```diff
@@ -16,7 +16,8 @@ partial class Program
         public const string SourceUrl = "https://github.com/Microsoft/Git-Credential-Manager-for-Windows";
         public const string EventSource = "Git Credential Manager";
 
-        internal const string ConfigAuthortyKey = "authority";
+        internal const string ConfigAuthorityKey = "authority";
+        internal const string ConfigLoginHintKey = "loginhint";
```

Comment:
Please look at "Docs/Configuration.md" and add an entry there on how this works.

The "Docs/Automation.md" might deserve to have this information as well. I'll let you decide.